frontend/src/core/editor/components/connectedEditor.css

```diff
@@ -3,7 +3,6 @@
     border-bottom: 1px solid #5e6475;
     display: flex;
     flex-direction: column;
-    height: calc(20% + 60px);
     min-height: 160px;
 }
```

Comment:
Since we no longer start with a calculated height, should this value be increased?

Reply:
I'd leave that for now. I think editor is mostly too big anyways, and it still fits 3 lines in the case of rich FTL editor.